<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Jugador</class>
 <widget class="QWidget" name="Jugador">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Jugadores</string>
  </property>
  <widget class="QPushButton" name="botonGuardar">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>530</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonBorrar">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>530</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Borrar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Jugadores</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nombre_text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Nombre:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>521</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="LimpiarCampos">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>530</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Limpiar Campos</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_genero">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>470</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Genero:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ApellidoPaterno_text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Apellido Paterno:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ApellidoMaterno_text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Apellido Materno:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numero_text">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>370</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Numero: (2 digitos)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="telefono_text">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>420</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Celular: (10 digitos)</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_equipo">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Equipo:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_categoria">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Categoria:</string>
   </property>
  </widget>
  <widget class="QLabel" name="equipo_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>521</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(84, 84, 84);
border-radius:3px;
border-color: rgb(29, 29, 29);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="id_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Id:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
